Vwitter/View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,69 @@
+//
+//  ComposeViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "ComposeViewController.h"
+
+@interface ComposeViewController () <UITextViewDelegate>
+@property (weak, nonatomic) IBOutlet UITextView *ventContent;
+@property (weak, nonatomic) NSString *placeholderText;
+
+@end
+
+@implementation ComposeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.ventContent.delegate = self;
+    self.placeholderText = @"Let it out!";
+    self.ventContent.text = self.placeholderText;
```

Comment:
I could be wrong here but I don't think this would be the best way to do a placeholder with a `UITextView`. No idea why UITextView doesn't support placeholder text like `UITextField` does....but nonetheless...

Typically I would achieve this by putting a UILabel "underneath" the `UITextView`, and use the `UITextViewDelegate` methods to hide/unhide it depending on whether the user has entered/erased the text in the text view.

It looks however like there is [a cocoapod you can install](https://github.com/devxoul/UITextView-Placeholder) that will handle it for you pretty easily, it basically adds what is called a "category" to the `UITextView` class that adds a placeholder. In ObjC, a category basically lets you add new methods/properties to a class that already exists. This would be a good recommendation for future problems as well, you can often find an easy way to do something by just installing it as a cocoapod/dependency to your project, especially minor stuff like this has almost always been solved before by someone

Reply:
Installed and implemented the cocoapod and it works! Will know to consult open source cocoapods/libraries for the future :)